### PR TITLE
Change tutorials template

### PR DESCRIPTION
**Pull request by gleybersonandrade**
Tutorials template has been changed to kytos sphinx theme.

### CODE REVIEW

File: tutorials/conf.py [hunk 1]
```diff
@@ -373,10 +374,12 @@
...
-intersphinx_mapping = {
-        'python-openflow': ('https://docs.kytos.io/python-openflow', None),
-        'kytos': ('https://docs.kytos.io/kytos', None),
-        'python': ('https://docs.python.org/3', None)}
+# intersphinx_mapping = {
+#         'python-openflow': ('https://docs.kytos.io/python-openflow', None),
+#         'kytos': ('https://docs.kytos.io/kytos', None),
+#         'python': ('https://docs.python.org/3', None)}
+
+intersphinx_mapping = {}
```
Comment (hdiogenes) [inline tutorials/conf.py]: Is this tested? We really don't need the intersphinx_mapping?

File: tutorials/napps/index.rst [hunk 2]
```diff
@@ -1,17 +1,7 @@
...
 
 If you have any doubt, do not hesitate to ask the Kytos community at our IRC
-Channel #kytos at freenode.net.
-
-.. toctree::
-  :maxdepth: 1
-  :hidden:
-
-  create_your_napp
+Channel #kytos at freenode.net.
```
Comment (hdiogenes) [inline tutorials/napps/index.rst]: This is outdated. Please update to https://kytos.slack.com/

**Review by hdiogenes — CHANGES_REQUESTED** [review]

File: tutorials/conf.py (hunk 1 above)
Comment (gleybersonandrade) [reply tutorials/conf.py]: Probably I made these changes in first time I tested this repo. Sorry @hdiogenes, on the next commit I undo these changes.

File: tutorials/napps/index.rst (hunk 2 above)
Comment (gleybersonandrade) [reply tutorials/napps/index.rst]: It's up to date now. Thanks @hdiogenes!

**Review by hdiogenes — APPROVED** [review]

**Pull request merged**